manifests/init.pp

```diff
@@ -59,6 +60,7 @@
   $config_dir_mode      = $filebeat::params::config_dir_mode,
   $config_dir           = $filebeat::params::config_dir,
   $config_file_mode     = $filebeat::params::config_file_mode,
+  $directory            = $filebeat::params::directory,
```

Comment:
It looks like `config_dir` might have been a bad parameter name to use for the prospector configuration directory, and would have been better served here.

Before I start looking for ways to change this around without breaking anything, is this only being used to create the configuration base directory?